src/Math-Complex/PMComplex.class.st

```diff
@@ -694,14 +707,15 @@ PMComplex >> sinh [
 
 { #category : #'mathematical functions' }
 PMComplex >> sqrt [
+
 	"Return the square root of the receiver with a positive imaginary part."
 
 	| u v |
-	(imaginary = 0 and: [real >= 0])
-		ifTrue:	[^self class real: real sqrt imaginary: 0].
+	(imaginary = 0 and: [ real >= 0 ]) ifTrue: [ 
+		^ self class real: real sqrt imaginary: 0 ].
 	v := (self abs - real / 2) sqrt.
 	u := imaginary / 2 / v.
-	^self class real: u imaginary: v
+	^ self class real: u imaginary: v
```

Comment:
These may be formatting improvements by Pharo when I inlined the `isReal` method